src/main/java/com/mhcure/logmerge/LogAggregatorApplication.java

```diff
@@ -90,7 +91,7 @@ private boolean isMergingRequired() {
                 MhFileAggregatorHelper.printToConsole(MhMessagePropertiesFileReader
```

Comment:
line no 85 u can move hardcoded string to constants. similarly do it for all files.

Reply:
Done